src/classes/fflib_Domain.cls

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c), FinancialForce.com, inc
```

Comment:
Can you confirm that we need to keep the FinancialForce copyright here?

Reply:
Well, I'd like to eventually include the entire library, though we are not doing that for this batch of work.  So I'd like to keep the file the same as it is in the AEP 2.0 library. The license says in the second bullet that it must be included in usages.  Is there anyone in particular that I need to check with?